include/zephyr/drivers/mfd/ad5592.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2023 Grinn
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_MFD_AD5592_H_
+#define ZEPHYR_INCLUDE_DRIVERS_MFD_AD5592_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/device.h>
+
+#define AD5592_REG_SEQ_ADC 0x02U
+#define AD5592_REG_ADC_CONFIG 0x04U
+#define AD5592_REG_LDAC_EN 0x05U
+#define AD5592_REG_GPIO_PULLDOWN 0x06U
+#define AD5592_REG_READ_AND_LDAC 0x07U
+#define AD5592_REG_GPIO_OUTPUT_EN 0x08U
+#define AD5592_REG_GPIO_SET 0x09U
+#define AD5592_REG_GPIO_INPUT_EN 0x0AU
+#define AD5592_REG_PD_REF_CTRL 0x0BU
+
+#define AD5592_EN_REF BIT(9)
+
+#define AD5592_PIN_MAX 8U
+
+/**
+ * @brief AD5592 MFD driver APIs
+ * @defgroup mfd_ad5592_apis ad5592
+ * @{
+ */
```

Comment:
this looks like a top-level group, you should add it to mfd (or create it if not present, see doc/_doxygen/groups.dox)

Reply:
Do you mean something like that?

<pre>
/**
 * @defgroup mdf_interface_ad5592 MFD AD5592 interface
 * @ingroup mfd_interfaces
 * @{
 */
</pre>

Reply:
yes

Reply:
Done.